Sources/Networking/Operations/NetworkOperation.swift

```diff
@@ -42,9 +42,9 @@ class NetworkOperation: Operation {
             return self._isExecuting.value
         }
         set {
-            self.willChangeValue(for: \.isExecuting)
+            self.willChangeValue(forKey: #keyPath(NetworkOperation.isExecuting))
             self._isExecuting.value = newValue
-            self.didChangeValue(for: \.isExecuting)
+            self.didChangeValue(forKey: #keyPath(NetworkOperation.isExecuting))
```

Comment:
let's definitely leave a note here explaining why it's important to use keyPath here, there's no way a maintainer would guess it if they're not familiar

Reply:
we can even link to the discussion with a rebrandly link or something

Reply:
Good call.

Reply:
Done.